### PR TITLE
Add column for collection to tools, organisation and contributors.

### PR DESCRIPTION
**Pull request by koechkevin**
## Description

There's a column we missed during our implementation called collection, this PR adds that column to Ecosystem Settings, Airtable process Data and an ability to filter by the same column via block settings.

Since we cannot name a column as `collection` in payload, I have used `{entity}Collection` naming standard.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
Ola @kelvinkipruto 

1. Where are we showing the fitler?
2. It looks broken to me (i.e. appears on a new line below other filters). Shouldn't all align on one row (Do we have updated designs that include this filter?

**Comment by kilemensi** [comment]
> Ola @koechkevin
> 
>     1. Where are we showing the filter?
> 
>     2. It looks broken to me (i.e. appears on a new line below other filters). Shouldn't all align on one row (Do we have updated designs that include this filter?

**Comment by koechkevin** [comment]
> Ola @koechkevin
> 
> 1. Where are we showing the filter?
> 2. It looks broken to me (i.e. appears on a new line below other filters). Shouldn't all align on one row (Do we have updated designs that include this filter?

@kilemensi  I added this to blocks to select if it is required to appear as a filter.

image

**Comment by kilemensi** [comment]
> > Ola @koechkevin
> > 
> > 1. Where are we showing the filter?
> > 2. It looks broken to me (i.e. appears on a new line below other filters). Shouldn't all align on one row (Do we have updated designs that include this filter?
> 
> @kilemensi I added this to blocks to select if it is required to appear as a filter.
> 

Cool @koechkevin, my question was how do we prevent things like

image

i.e. selection spilling over multiple lines?

File: apps/charterafrica/src/lib/data/common/processPageContributors.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import formatDateTime from "@/charterafrica/utils/formatDate";
 import labelsPerLocale from "@/charterafrica/utils/translationConstants";
 
 const queryBuilder = (query) => {
-  const { search, location } = query;
+  const { search, location, contributorCollection } = query;
```
Comment (kilemensi) [inline apps/charterafrica/src/lib/data/common/processPageContributors.js:11]: Why no just `collection` (similar to just `location`)?

File: apps/charterafrica/src/components/EcosystemList/EcosystemFilter.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { ControlledSearchInput } from "@/charterafrica/components/SearchInput";
 
 const EcosystemFilter = React.forwardRef(function EcosystemFilter(props, ref) {
   const { onChange, searchPlaceholder, values, filterOptions = [] } = props;
-
+  const spans = Math.floor(9 / filterOptions.length);
```
Comment (kilemensi) [inline apps/charterafrica/src/components/EcosystemList/EcosystemFilter.js:11]: Add a comment to explain why `9`

File: apps/charterafrica/src/lib/data/common/processPageContributors.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ import formatDateTime from "@/charterafrica/utils/formatDate";
 import labelsPerLocale from "@/charterafrica/utils/translationConstants";
 
 const queryBuilder = (query) => {
-  const { search, location } = query;
+  const { search, location, classification } = query;
```
Comment (kilemensi) [inline apps/charterafrica/src/lib/data/common/processPageContributors.js:11]: The data team confirmed collection === classification?

**Review by kilemensi — APPROVED** [review]

File: apps/charterafrica/src/lib/data/common/processPageContributors.js (hunk 3 above)
Comment (koechkevin) [reply apps/charterafrica/src/lib/data/common/processPageContributors.js:11]: @kilemensi 

From this [document](https://docs.google.com/document/d/1HDQBaOJD8XjtJfo-RR5s-TOdh1njC88ZmDqUVNeJwxg/edit?usp=sharing),
image

Comment (kilemensi) [reply apps/charterafrica/src/lib/data/common/processPageContributors.js:11]: Thanks @koechkevin ... If I understand the doc, `classification` is the _source_ for `collection`, it doesn't mean `classification` **is** `collection`. Similar to `DB` is the source for `Tool` but you can't say `DB` **is** `Tool`.

But anyways, let's get it over the line and push on academy.AFRICA.

**Pull request merged**